src/shared/bus-util.c

```diff
@@ -619,10 +619,6 @@ int bus_connect_user_systemd(sd_bus **_bus) {
         if (r < 0)
                 return sd_bus_default_user(_bus);
 
-        r = bus_check_peercred(bus);
```

Comment:
If you're OK with this approach...

I guess a follow up question is whether we should also drop the check from `bus_connect_system_systemd()`, given the `/run/systemd/private` socket is always owned by root (so the `ucred.uid != 0` check makes it always return OK), we could maybe drop it from there as well.